### PR TITLE
Fix looking for processes on osx

### PR DESCRIPTION
**Pull request by robvanmieghem**
Solves https://github.com/threefoldtech/js-sdk/issues/216

### CODE REVIEW

File: jumpscale/sals/process/__init__.py [hunk 1]
```diff
@@ -569,8 +569,7 @@ def is_port_listenting(port):
     Returns:
         Bool: True if port is used else False
     """
-    pcons = [proc for proc in psutil.net_connections() if proc.laddr.port == port and proc.status == "LISTEN"]
```
Comment (abom) [inline jumpscale/sals/process/__init__.py:572]: I didn't see this, but this would just break it on other platforms.

Comment (abom) [reply jumpscale/sals/process/__init__.py:572]: Will revert until we find another solution that's fit for both.

Comment (robvanmieghem) [reply jumpscale/sals/process/__init__.py:572]: I bypassed the error on the line you see with a hack to get to the other error, also, why would ignoring the FileNotFound Error break the routine on other platforms? For me it only gave an error on iTerm2

Comment (robvanmieghem) [reply jumpscale/sals/process/__init__.py:572]: there is an issue for the  error on the line you marked  as well: https://github.com/threefoldtech/js-sdk/issues/215

Comment (robvanmieghem) [reply jumpscale/sals/process/__init__.py:572]: damned, I indeed deleted that line to bypass an error there

Comment (robvanmieghem) [reply jumpscale/sals/process/__init__.py:572]: should I make a pull request just for https://github.com/threefoldtech/js-sdk/issues/216 ?

Comment (abom) [reply jumpscale/sals/process/__init__.py:572]: oh, sure!
please do.

Comment (robvanmieghem) [reply jumpscale/sals/process/__init__.py:572]: https://github.com/threefoldtech/js-ng/pull/415

**Pull request merged**